app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -321,34 +321,11 @@ ListingController = (
     return type unless labelMap[type]
     return labelMap[type][modifier]
 
-  $scope.priorityLabel = (priority, modifier) ->
```

Comment:
@callaghanc you mentioned on the SF ticket that you thought this might be no longer necessary. I checked and it looks like on production we were displaying the impair*ed* version even though this function looks like it was trying to make it display the impair*ments* version. 

I'm also not sure which we would like to display, but based on this function is looks like we were trying for impair*ments*.

Reply:
my comment mentioned that the "impaired" language in this function should not be necessary, since we're switching to "impairments" to match the new language in salesforce. i haven't looked closely enough into this function to see exactly how it handles the language - lmk if you would like a pair on this

Reply:
sure! maybe pairing could help!